HLTrigger/Configuration/python/HLT_75e33_timing_cff.py

```diff
@@ -2,6 +2,16 @@
 
 fragment = cms.ProcessFragment("HLT")
 
+### ACCELERATORS, in case they are needed
+## Load explicitly
+# One ProcessAccelerator for each accelerator technology
+fragment.load("Configuration.StandardSequences.Accelerators_cff")
+
+# And one ProcessAccelerator for Alpaka
+# (eventually to be absorbed to Accelerators_cff)
+fragment.load("HeterogeneousCore.AlpakaCore.ProcessAcceleratorAlpaka_cfi")
+
+
```

Comment:
```suggestion
```
This is already loaded by [Configuration/StandardSequences/python/Accelerators_cff.py](https://github.com/cms-sw/cmssw/blob/master/Configuration/StandardSequences/python/Accelerators_cff.py#L8)